cli/tests/e2e/test_dependency_aware_rules.py

```diff
@@ -154,6 +154,15 @@
             "rules/dependency_aware/python-poetry-sca.yaml",
             "dependency_aware/manifest_parse_error",
         ),
+        (
+            "rules/dependency_aware/nuget-sca-simple.yaml",
+            "dependency_aware/nuget",
+        ),
+        # This test intentionally runs poetry rules against C# to check that scan runs correctly and does not produce findings
+        (
+            "rules/dependency_aware/python-poetry-sca.yaml",
+            "dependency_aware/nuget",
+        ),
```

Comment:
Not adding more tests until I decide on what direction to take wrt same dependency instances across different target frameworks.

Reply:
I believe you said this got resolved?

Reply:
There's a little bit of contention around how we want to solve this, so I'm setting up a project in Snyk to check its behaviour 🥼 🐀